cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,217 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events attributes are expressed in
+the [Avro 1.9.0 Specification][avro-spec].
+
+This differs from the [Avro format](avro-format.md) in that:
+
+- It is optimized for performance, preferring a more compact representation.
+- Only supports spec version 1.0 (any changes to spec version, requires 
+  changes to Avro schema, which changes the finger-print, 
+  breaking compatibility).
+- Does not natively support JSON (JSON can be straight-forwardly serialized 
+  to bytes and this was therefore not considered neccessary).
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Data](#3-data)
+4. [Transport](#4-transport)
+5. [Examples](#5-examples)
+
+## 1. Introduction
+
+[CloudEvents][ce] is a standardized and protocol-agnostic definition of the
+structure and metadata description of events. This specification defines how the
```

Comment:
Again, this needs to be less definitive if there are two alternative formats.

Reply:
fixing